_data/people/davidlange6.yml

```diff
@@ -25,3 +25,19 @@ presentations:
     meeting: "Hong Kong IAS Mini-workshop: Experiment / Detector - Software and Physics Requirements for e+e- Colliders"
     meetingurl: http://iasprogram.ust.hk/hep/2020/activities.php
     focus-area: ia
+  - title: "Challenges and Evolution of the LHC computing model for HL-LHC"
```

Comment:
All talks are required to at least have a URL

Reply:
hum, none exists.

Reply:
guess I can dump slides on a webpage